go.mod

```diff
@@ -0,0 +1,56 @@
+module github.com/lyft/flytestdlib
+
+go 1.13
+
+require (
+	cloud.google.com/go v0.47.0 // indirect
+	github.com/Azure/azure-sdk-for-go v10.2.1-beta+incompatible // indirect
+	github.com/Azure/go-autorest/autorest v0.9.2 // indirect
+	github.com/aws/aws-sdk-go v1.25.16
+	github.com/benlaurie/objecthash v0.0.0-20180202135721-d1e3d6079fc1
+	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/cespare/xxhash v1.1.0 // indirect
+	github.com/cheekybits/is v0.0.0-20150225183255-68e9c0620927 // indirect
+	github.com/coocood/freecache v1.1.0
+	github.com/dnaeon/go-vcr v1.0.1 // indirect
+	github.com/ernesto-jimenez/gogen v0.0.0-20180125220232-d7d4131e6607
+	github.com/fatih/color v1.7.0
+	github.com/fatih/structtag v1.1.0
+	github.com/fsnotify/fsnotify v1.4.8-0.20191012010759-4bf2d1fec783
+	github.com/ghodss/yaml v1.0.0
+	github.com/golang/groupcache v0.0.0-20191002201903-404acd9df4cc // indirect
+	github.com/golang/protobuf v1.3.2
+	github.com/graymeta/stow v0.0.0-20190522170649-903027f87de7
+	github.com/hashicorp/golang-lru v0.5.3
+	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
+	github.com/magiconair/properties v1.8.1
+	github.com/mattn/go-colorable v0.0.9 // indirect
+	github.com/mattn/go-isatty v0.0.10 // indirect
+	github.com/mitchellh/mapstructure v1.1.2
+	github.com/ncw/swift v1.0.49-0.20190728102658-a24ef33bc9b7 // indirect
+	github.com/pelletier/go-toml v1.5.0 // indirect
+	github.com/pkg/errors v0.8.1
+	github.com/prometheus/client_golang v1.0.0
+	github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4 // indirect
+	github.com/prometheus/common v0.7.0 // indirect
+	github.com/prometheus/procfs v0.0.5 // indirect
+	github.com/satori/uuid v1.2.0 // indirect
+	github.com/sirupsen/logrus v1.4.2
+	github.com/spf13/afero v1.2.2 // indirect
+	github.com/spf13/cobra v0.0.5
+	github.com/spf13/jwalterweatherman v1.1.0 // indirect
+	github.com/spf13/pflag v1.0.5
+	github.com/spf13/viper v1.3.2
```

Comment:
Can you double check this includes the SHA from Gopkg.toml?

Reply:
I think this commit is included since v1.3.0
https://github.com/spf13/viper/commit/ae103